src/js/core/widget/core/VirtualListview.js

```diff
@@ -509,6 +512,11 @@
 				self._currentIndex = index;
 			}
 
+			function getFirstElementChild(element) {
+				return (element.firstElementChild.tagName === "LI") ?
+					element.firstElementChild : element.firstElementChild.nextElementSibling;
```

Comment:
Please consider returning the first element which is actually "LI" element. Second element also can be something different than "li".
This may cause problems in the future in case if listview widget structure will change.

Reply:
Done.